test/test_circle.py

```diff
@@ -5,6 +5,12 @@
 from typing import List
 import unittest
 
+from test.testing_common import (
```

Comment:
What's your motivation for changing the order there? I usually order such that the modules that are closest to the current code are coming last.

Reply:
Yes, so, both `lndmanage.settings` and `test.testing_common` have side-effects on import where they attempt to read/create the config file, and can error if this fails.

Before these changes, running the tests in a clean environment can fail due to this.

The idea here is that `test.testing_common` ensure that this does not happen by ensuring consistency and a default for `LNDMANAGE_HOME`. So `testing_common` needs to be imported before `lndmanage.settings`.

I think there was some discussions elsewhere on how it would be desirable to not have side-effects-by-import, which can be good to address down the line, but at least these changes (#d788ceb7d05214694561156522ffd624666f218b in isolation) will ensure that the test suite doesn't interact with any existing config on the system.

Reply:
Ok, I see